src/api-calls.js

```diff
@@ -0,0 +1,11 @@
+export default (path) => {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${path}`)
```

Comment:
I am considering having the url be a variable that we can import too especially since we have to type it so many times in the cyrpress testing